llvm/tools/llvm-dis-fuzzer/CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 set(LLVM_LINK_COMPONENTS
   BitReader
+  Core
+  Support
```

Comment:
Why is this necessary? They should already be transitive deps of BitReader: https://github.com/llvm/llvm-project/blob/be10070f91b86a6f126d2451852242bfcb2cd366/llvm/lib/Bitcode/Reader/CMakeLists.txt#L14

Reply:
To be frank, I don't have a strong enough understanding of the LLVM build system to properly articulate an answer. 

This change was necessary to get my local build working. 

My local build uses `-DBUILD_SHARED_LIBS=ON`.  When I tested `-DBUILD_SHARED_LIBS=OFF`, this change was not needed to use `-DLLVM_USE_SANITIZE_COVERAGE=ON`.

So, perhaps the transitive dependencies are not automatically added when using `-DBUILD_SHARED_LIBS=ON`. Does this seem reasonable to you @nikic?

Reply:
Hm yeah, after looking at the cmake code a bit I think we make the dependencies of shared objects private.